### PR TITLE
Added a Tilting Maze Game Javascript Challenge

### PR DESCRIPTION
**Pull request by 5h0ov**
I have added an Unique Tilting Maze game which has been categorized in 'Hard'  Difficulty under Javascript Challenges.

## Working Video:
tilt

Kindly review this PR and merge it under hacktoberfest 2023.

### CODE REVIEW

File: native/machine-coding/tilting-maze/styles.css [hunk 1]
```diff
@@ -0,0 +1,204 @@
+body {
+  --background-color: #ede6e3;
+  --wall-color: #36382e;
+  --joystick-color: #210124;
+  --joystick-head-color: rgba(255, 0, 0, 0.91);
+  --ball-color: #a90af2;
+  --end-color: #8b91dc;
+  --text-color: #210124;
+
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
```
Comment (sadanandpai) [inline native/machine-coding/tilting-maze/styles.css:10]: import core styles like other challenges and not override HTML, BODY without necessity

File: native/machine-coding/tilting-maze/styles.css [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+  position: absolute;
+}
+
+.wall::before 
+{
+  top: -5px;
+}
+
+.wall::after 
+{
```
Comment (sadanandpai) [inline native/machine-coding/tilting-maze/styles.css:192]: follow the consistent syntax

File: native/machine-coding/tilting-maze/script.js [hunk 3]
```diff
@@ -0,0 +1,680 @@
...
+  { column: 7, row: 7, horizontal: false, length: 2 },
+
+  // Vertical lines after the 8th column
+  { column: 8, row: 2, horizontal: false, length: 1 },
+  { column: 8, row: 4, horizontal: false, length: 2 },
+
+  // Vertical lines after the 9th column
+  { column: 9, row: 1, horizontal: false, length: 1 },
+  { column: 9, row: 5, horizontal: false, length: 2 }
+].map((wall) => ({
```
Comment (sadanandpai) [inline native/machine-coding/tilting-maze/script.js:204]: can we simplify this and move it to a different file if needed

**Review by sadanandpai — APPROVED** [review]

**Review by sadanandpai — CHANGES_REQUESTED** [review]
Added the comments

File: native/machine-coding/tilting-maze/styles.css (hunk 1 above)
Comment (5h0ov) [reply native/machine-coding/tilting-maze/styles.css:4]: done

File: native/machine-coding/tilting-maze/styles.css (hunk 2 above)
Comment (5h0ov) [reply native/machine-coding/tilting-maze/styles.css]: made it consistent sir could u take a look

File: native/machine-coding/tilting-maze/script.js (hunk 3 above)
Comment (5h0ov) [reply native/machine-coding/tilting-maze/script.js]: i separated the function that is used to draw the  walls for the maze in a separate JS file.

Is that ok?

**Review by sadanandpai — APPROVED** [review]

**Pull request merged**